package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dynamods/dynamo-home",
-  "version": "1.0.13",
```

Comment:
why are we changing the version here? the version on apphome repo is still 1.0.13?

Reply:
oh sorry i thought this was on dynamorepo.